libs/core/src/lib/nested-list/nested-item/nested-item.directive.spec.ts

```diff
@@ -57,6 +58,7 @@ describe('NestedItemDirective', () => {
     let nestedItemListDirective: NestedItemDirective;
     let emptyItemDirective: NestedItemDirective;
     let fixture: ComponentFixture<TestNestedContainerComponent>;
+    let itemService: NestedItemService;
```

Comment:
Shouldn't itemService be a const?

Reply:
It's overwritten below. There is only declaration